### ISSUE TITLE
Installed clockgen and gbs board is not responding

### ISSUE THREAD

**Issue by israpps** [ramapcsx2/gbs-control#439]
As title says.
Asked a friend to solder the clockgen (he makes boards for a kinda big factory, so he surely knows what he did)

got it back today, plugged it in and did not get any signal

opened web UI and enabled developer mode to see the log

touching any item displays on log
```
GBS board not responding!
```

any ideas?

will attach pics of the board in a few mins

In the meantime, ill try updating the firmware to the newer builds, as the clockgen library has been updated

**Comment by israpps** [comment]
this is a duplicate of #291 buuut. there are no messages there that point into the solution

**Comment by israpps** [comment]
![IMG_20230313_190937472](https://user-images.githubusercontent.com/57065102/224843589-f0af51a8-f5cd-4f93-9b1a-e1ebd0e9c335.jpg)
![Uploading IMG_20230313_190958270.jpg…]()

**Comment by israpps** [comment]
@ramapcsx2 any idea?

**Comment by ramapcsx2** [comment]
Not really.. The message means that the GBS chip didn't respond on the I2C bus.
It could be power related, signals conflicting (SDA/SCL swap or so), etc.

**Comment by israpps** [comment]
![IMG_20230313_095423026](https://user-images.githubusercontent.com/57065102/225329932-febb1bb3-1d9c-4ebd-9844-1b6a9ce6bcf5.jpg)

**Comment by ramapcsx2** [comment]
Okay, try to remove the clock gen SDA and SCL wires. If it then works, you know that the I2C bus is getting bad with the clock gen on it, and you can look into why that might be.

**Comment by israpps** [comment]
> Okay, try to remove the clock gen SDA and SCL wires. If it then works, you know that the I2C bus is getting bad with the clock gen on it, and you can look into why that might be.

could it be swapped SDA/SCL (either by mistake of who installed it or by bad labeled clockgen), I bought it here on argentina, so its 99% not original xd.

**Comment by israpps** [comment]
Tested the GBSControl UI clockgen related stuff and it seem to respond

```console
user command f at settings source 10, custom slot 65, status 0
GBS board not responding!
user command s at settings source 0, custom slot 65, status 0
GBS board not responding!
user command r at settings source 5, custom slot 65, status 0
Deinterlacer: Motion Adaptive
user command X at settings source 5, custom slot 65, status 0
ExternalClockGenerator disabled
user command h at settings source 5, custom slot 65, status 0
GBS board not responding!
user command g at settings source 1, custom slot 65, status 0
GBS board not responding!
user command f at settings source 3, custom slot 65, status 0
GBS board not responding!
user command 5 at settings source 0, custom slot 65, status 0
FTL on
Active FrameTime Lock enabled, adjusting external clock gen frequency
user command g at settings source 0, custom slot 65, status 0
GBS board not responding!
user command f at settings source 3, custom slot 65, status 0
GBS board not responding!
```
@ramapcsx2

**Comment by YoImLee** [comment]
"FTL on - Active FrameTime Lock enabled, adjusting external clock gen frequency" Is what you want to see on recent versions of gbs control. It is better to leave frametime lock enabled now even with the clock gen installed.

In the first picture, the pin next to pin 40 for the clock generator input is tinned and may be bridged. There might also be a bridge in the second picture at the debug pin. Kind of hard to tell though in both cases, I might be wrong.

**Comment by israpps** [comment]
> "FTL on - Active FrameTime Lock enabled, adjusting external clock gen frequency" Is what you want to see on recent versions of gbs control. It is better to leave frametime lock enabled now even with the clock gen installed.
> 
> In the first picture, the pin next to pin 40 for the clock generator input is tinned and may be bridged. There might also be a bridge in the second picture at the debug pin. Kind of hard to tell though in both cases, I might be wrong.

Issue must be related to clockgen install...

As I used the board without it for months.

And the esp setup hasn't been changed.

**Comment by israpps** [comment]
Tested with multimeter...

Clockgen wire that goes into scaler chip is not bridged (measured continuity between CLK0 and the pin next to the one soldered)

Debug pin on GBS chip is not bridged.

To be sure, I checked continuity of SDA/SCL from clockgen pin up to their esp8266 equivalents. Everything ok there

Only possible clue is power.

The esp8266 is taking power from the 3.3v of GBS board

**Comment by ramapcsx2** [comment]
Can you please unsolder the 2 clock gen SDA/SCL wires and check what happens then?

**Comment by israpps** [comment]
> Can you please unsolder the 2 clock gen SDA/SCL wires and check what happens then?

Same issue

There is a short between r11 and r37... Is that normal?

**Comment by israpps** [comment]
Also, made esp8266 take power from PSU

**Comment by israpps** [comment]
![IMG_20230317_120923586~2](https://user-images.githubusercontent.com/57065102/225945819-146daa08-3795-41fa-9608-839cb459c66d.jpg)
I have continuity testing between red and yellow dots

**Comment by israpps** [comment]
Disconnected whole clock

GBS board is still dead🙄

**Comment by israpps** [comment]
Anyone can help troubleshooting this?

Or should I buy a new GBS board instead

**Comment by ramapcsx2** [comment]
So since it's still dead with the I2C wires for the clock gen removed, it means that when installing, you accidentally broke something else. Find that "else" next.
You mention a short between R11 and R37. One side of them would be correct (the Vcc side), the other must not be shorted.
If you probe the 2 sides and both are shorted, this is your problem.

**Comment by israpps** [comment]
not sure what else to test

tried everything that came to my mind...
Not sure what else to do!

**Comment by ser51** [comment]
I have been successfully using GBS 8220 + ESP8266 + SI5351 for a year. But since yesterday, the device began to give me "GBS board not responding!". Checked all wiring. All OK. Disconnected SI5351, the problem did not go away. My problem was solved only by replacing the Y1 quartz with 27.000. Perhaps someone will help.
